### ISSUE TITLE
Soap dispensers

### ISSUE THREAD

**Issue by iteles** [dwyl/home#174]
We have long since agreed that the most hygienic form of soap dispenser is a foaming soap dispenser with a motion sensor. 

We have been testing a couple of brands since 2019 and without a doubt Laopao has been the best one. As long add it is kept clean, it dispenses a good amount of foam quickly and without fail. 

It is also more eco-friendly because it uses less hand soap. The _ideal _ in ecological terms would be use _bars_ of soap which is not as hygienic but I think we can test shaving some down to make own foaming soap solution. 

@nelsonic I’m ready to pull the trigger on buying this brand for each bathroom unless there is a different system you want to consider?
I’ve already scoped out where to buy it and can quickly get this done.

**Comment by nelsonic** [comment]
@iteles thank you for opening this issue to capture our research, testing and decision on the foam soap dispensers. 👍🏻

While I _like_ the **Laopao** dispenser, I have two issues with it:
1. The tank is *tiny* (250ml) so it requires regular refilling. (And with more people in the house, possibly _Weekly_ refilling…)
2. It’s a bit “plasticky” and has the brand **Laopao** very visible on the front.

Obviously Amazon have jacked up the price 30% since we bought ours (pre-pandemic) but the reviews reflect our experience with the product.

https://www.amazon.com/dp/B07RHS2XG2/
![C2560D97-743C-4C33-A407-53093C8DF80C](https://user-images.githubusercontent.com/194400/113408591-867b9680-93a7-11eb-958d-e6fd4d00b2ea.png)

If there was no other option, I would go with the Laopao. But I feel like we should consider other options first. 


FENRIR Soap Dispenser Automatic 350ml, No Touch Soap Dispenser, Foaming Soap Dispenser Pump, Volume IR Sensor Touchless Hand Soap Dispenser for Bathroom, Kitchen, Office, Home https://www.amazon.co.uk/dp/B08NDBY2X8/
![CA0E6E05-3C51-484C-B0A4-7D92E0019C2E](https://user-images.githubusercontent.com/194400/113407811-01dc4880-93a6-11eb-98ad-27464e478dad.png)

The “fenrir” model has good reviews, a 350ml tank and no visible branding on the front. Annoyingly, it’s only available in the U.K! 🤦‍♂️
so unless we want to pay import duties to PT we need to consider other options. 💭

the selection on Amazon ES is *poor*:
https://www.amazon.es/s?k=automatic+foam+Soap+Dispenser&s=review-rank

but this looks like the best one: 
https://www.amazon.es/dp/B08MXL642R/

![858650F7-C7F1-4276-8A0A-21F7149B35EA](https://user-images.githubusercontent.com/194400/113410250-e162bd00-93aa-11eb-8227-f3fd3aa606c8.png)

I like the look of this one:      https://www.amazon.de/-/en/Dispenser-Automatic-Waterproof-Infrared-Non-Contact/dp/B06XWHMP9Q
![E9893F9C-BC4D-409C-816A-60D24EE5B05E](https://user-images.githubusercontent.com/194400/113411071-cbee9280-93ac-11eb-961c-587bcfa770df.png)

350ml transparent tank and no branding. 💭

**Comment by nelsonic** [comment]
@iteles how much are the Laopao dispensers including shipping? 
The official Laopao store on AliExpress has them for **$17.45**:  https://www.aliexpress.com/item/4001167064617.html
![image](https://user-images.githubusercontent.com/194400/113434608-2d2c5b00-93d9-11eb-9f3d-0fea6aca767a.png)
but the shipping to PT is extortion. **$10.29** 🤦 

If we didn't need so many of them, I would totally get the **simplehuman** one: https://www.simplehuman.com/products/foam-sensor-pump
![image](https://user-images.githubusercontent.com/194400/113435824-7d0c2180-93db-11eb-8761-5f9606b03c42.png)
But $60 is a bit over-priced if we need a few of them ... 💸 🔥 

This one on Amazon ES doesn't have any visible branding, has a transparent 350ml tank and appears to work well in the customer review videos: https://www.amazon.es/Dispensador-Automatico/dp/B08P59L6LP
![image](https://user-images.githubusercontent.com/194400/113435633-21419880-93db-11eb-82b2-fe547a34f2b2.png)

**Comment by nelsonic** [comment]
Reasonably priced €14.99 (including VAT), 350ml transparent tank, no visible branding and free shipping:
https://www.amazon.es/Geekper-Dispensador-dispensador-autom%C3%A1tico-el%C3%A9ctrico/dp/B08863PNVW
![image](https://user-images.githubusercontent.com/194400/113436271-539fc580-93dc-11eb-888c-dd6cb46ed383.png)

> Note: the only reason I'm limiting my search to Amazon ES is because we know it will actually arrive and we won't be charged extra tax/import duties. 💭 But if you have a reliable alternative supplier, please go for it. 👍

**Comment by iteles** [comment]
I’m not bothered about the branding, we can cover that with home branding at a later date. 

The tank issue is a thing though, I’ve never noticed it because you refill them 🙈
How often do you estimate it would have to be refilled on a full house of people in our busiest bathroom?

Yes, unfortunately, ALL foaming soap dispensers I’ve found are shipped in from the other side of the world 🙄

The Laopao one I found on a ‘limited time special’ for the next 13 hours (it was just under two days when I opened this issue so it tracks):
https://m.pt.aliexpress.com/item/1005002296884215.html

![642DF3EA-C9C2-4926-8A4F-53C3A79875B5](https://user-images.githubusercontent.com/4185328/113436378-88138180-93dc-11eb-8038-0d90021389a6.png)


BUT having said all this, the one you saw on Amazon es with the bigger tank and no branding is much more in keeping with our bathrooms (I never liked the blue tank much for our decor). So we should just get those instead to try 👌🏻

**Comment by nelsonic** [comment]
Yeah, AliExpress has these time-based discounts much like Udemy https://github.com/dwyl/learn-flutter/issues/62 🙄 ...
The **`€4.58`** shipping brings the total price (**`€18.50`**) up in-line with the Amazon ones linked to above and there's always the danger of it being stopped by customs (for ages) if we order more than one because the box is quite bulky ... 💭 
Also the thing to know about AliExpress is that they often don't have _flat_ rate shipping; the price goes _up_ the more you order because they have different package sizes depending on quantity. 💭

**Comment by iteles** [comment]
Whether we had ordered from AliExpress or from Amazon, we’d have had to order them as individual items so as not to be hit by ridiculous customs charges which means more packaging and more waste. 

Glad we’re ordering the ones from Amazon.és to try out 👌🏻

**Comment by nelsonic** [comment]
Hey Darryl, add foam soap dispenser to our shopping list.
https://www.cc.com/video/vvpddx/south-park-hey-darryl-hey-darryl
😜

**Comment by nelsonic** [comment]
@iteles this is the one I recently bought: https://www.amazon.es/-/pt/dp/B09CPLYKX5
<img width="1458" alt="amazon-soap-dispenser" src="https://user-images.githubusercontent.com/194400/139534360-0fd1f44f-0e47-4ad4-bbd9-7c8b7e483e25.png">

It's been in the main floor bathroom for the past week, what have you thought of it?
Can/should I add another to our next AMZN ES order for the kitchen sink?

**Comment by iteles** [comment]
I still prefer the original ones we bought as the stream of foam is much more consistent and I'm really not a fan of the USB charging. It's MUCH more of a faff than using rechargeable batteries.

**Comment by nelsonic** [comment]
while I *agree* that using rechargeable AA/AAA batteries would definitely be preferable than an internal (non-replaceable) battery, charging via Micro USB is perfectly fine because it takes less than 30 mins and last 2-3 months. It’s a 1300mAh battery pack and I suspect the unit will last several years. (3-7 depending on frequency of use)

The foam is consistent/reliable:
![77B180F9-E433-436C-B16A-073C7D1FCB7B](https://user-images.githubusercontent.com/194400/139542578-2ec3c50c-43fe-4284-9b34-4b3be4b2405f.jpeg)

Obvs I would prefer a Battery one, but considering that I’m the one who always changes the batteries on the others we have and it takes more *effort* than charging with a USB cable, I think these are “fine”.

**Comment by iteles** [comment]
Must have been teething problems then. 

Do you want to just determine how many we will need in total and buy the ones we need rather than dribs and drabs?

**Comment by nelsonic** [comment]
If we were to get more I’d buy another 4.
Got one to test.
The real question is do we think the design of this one is aesthetically “ok” for our needs?

**Comment by iteles** [comment]
Yes. 

Ideally we'd get one that suctions onto the basin but we can retrofit that. 

And we can cover the brand with a sticker of our own branding.

**Comment by iteles** [comment]
The new soap dispense has been working great! And it appears to  look less 'grubby' than the ones with the coloured 'tanks' 👍  More of those are welcome 🙌

**Comment by nelsonic** [comment]
@iteles glad you've found it to be good.

Sadly ... this model is now ***`Não disponível.`*** 🤦‍♂️ 
https://www.amazon.es/-/pt/dp/B09CPLYKX5 
<img width="1437" alt="soap-dispenser-no-longer-available" src="https://user-images.githubusercontent.com/194400/142854202-a33f4163-2501-4c6e-92ce-14166ae326a6.png">

this is becoming a bit of a (not funny) joke ...  😢 
Back to the research again ... 🙄

**Comment by iteles** [comment]
No, we don't need any more for now. Let's leave it and it may well come back into stock in the new year 👍

**Comment by iteles** [comment]
We have now tried several different names and models and the original Laopao is the only one still left standing 😅

Seeing as we have someone moving back from Asia in 2 months who has offered to bring some things back for us, should we consider just getting a bunch more?

**Comment by nelsonic** [comment]
@iteles if you can still _find_ the "original" **`LAOPAO`** one ... 🤷‍♂️ 
It's not available ("Não disponível") on Amazon ES: https://www.amazon.es/dp/B07RHS2XG2/
![image](https://user-images.githubusercontent.com/194400/179363476-01e8928a-772e-47ff-ba48-9d923dfbc8ef.png)
And I'm _really_ not confident that it will be the _exact_ same product from AliExpress ...
https://www.aliexpress.com/item/1005002296884215.html
![image](https://user-images.githubusercontent.com/194400/179363622-90144dd9-8b43-47ef-9825-96a5942732a6.png)
At **`€ 20,07`** it's not much of a "risk" in terms of price. 💭 

But to be honest with you I feel "burned" by all of the "budget" dispensers _including_ the **LAOPAO** ... 🔥 
They are all a pain to refill _consistently_ and I'm the only one who does it. 🙄 

The one thing that the **`LAOPAO`** has going for it is that it uses **`AA`** batteries not a built-in rechargeable:
![image](https://user-images.githubusercontent.com/194400/179363782-e06b56d7-7968-4c6f-9065-2e5c7ccb80e2.png)
We use IKEA Rechargeable https://github.com/dwyl/home/issues/209 batteries in ours and they last a 6+ months with regular use (e.g. 20 uses per day). I refill the soap once a month but the batteries keep going! 

## simplehuman `ST1062`

As noted above https://github.com/dwyl/home/issues/174#issuecomment-812612959 I'm keen on the **`simplehuman`** sensor pump foam: https://www.simplehuman.com/products/foam-sensor-pump
![image](https://user-images.githubusercontent.com/194400/179371960-90947f4b-e118-4dc0-a543-3a6059b6ae30.png)


Interesting how the price has gone up **`$10`** (**`17%`**!) in a year ... 📈 **`#inflation`**

I tried buying directly from the SimpleHuman ES 🇪🇸 site:
https://www.simplehuman.es/products/foam-sensor-pump?variant=39255458414656
![image](https://user-images.githubusercontent.com/194400/179372222-05dccd9f-e063-48b0-9f23-501a495a9435.png)

But was rejected as they don't want to deliver to PT.
![image](https://user-images.githubusercontent.com/194400/179372264-239606a2-8a5a-4bb9-96bc-dcdb1dc7a719.png)

We could ask my cuz [Shamus](https://www.linkedin.com/in/sean-eady/) if he doesn't mind us sending a couple to his address in Madrid and we bring them back with us. 

Honestly, I feel like the soap dispenser is such a daily use item that having a _really_ good one makes a ton of sense.
If we have to spend a bit more money and they last 3-4 times longer and are more consistent then it's a worthwhile investment IMO. 💭

**Comment by iteles** [comment]
I am honestly ***this close*** to getting good looking manual foaming soap dispensers that won't break and being done with this saga:
https://www.backtobasicsworld.com/product-page/dispensador-de-vidro-para-sabonete-l%C3%ADquido-ben-anna
<img width="910" alt="image" src="https://user-images.githubusercontent.com/4185328/179373223-72c27254-b708-4429-aa35-2fa8765e8627.png">

My gripe with the Simple Human ones is that they will _always_ look 'dirty' because the chrome (be it matte or shiny) will have water stains on it from every single use.

**Comment by nelsonic** [comment]
The Reviews for the simplehuman foam dispenser are that they don’t get dirty. Brushed stainless steel is remarkably resilient. And if that was a problem our ***chrome*** taps would be *disgusting*, which they aren’t. 

Manual soap dispensers are a fail when we have sensor taps. 👼don’t have the dexterity/strength them; they are [poorly] designed for adults.

**Comment by iteles** [comment]
Then speak to Sean and order 2 of the ones you like 👌🏻
